GoedeDoelenHelpen/Controllers/AuthenticationController.cs

```diff
@@ -251,6 +277,29 @@ public class AuthenticationController : Controller
             return new ActionResultModel { Success = false, Message = "Er is iets misgegaan." };
         }
 
+        [HttpGet("[action]")]
+        [ProducesResponseType(typeof(FaceBookHookedUp), 200)]
+        [ProducesResponseType(typeof(FaceBookNotHookedUp), 201)]
+        [Authorize]
+        public async Task<ActionResult<IFaceBookInfo>> FacebookInfo() {
+            ApplicationUser user = await this.GetApplicationUserAsync(_userManager);
+            if (user != null) {
+
+                List<FacebookRecord> userFBRecords = _context.FacebookRecords.Where((i) => i.ApplicationUserId == user.Id).ToList();
+
+
+                if (_context.FacebookRecords.Count() != 0 && userFBRecords.Count == 1) {
```

Comment:
Bestaat er een kans dat als de tweede true oplevert dat het alsnog niet doorgaat omdat de eerste false is?